### ISSUE TITLE
Should kadalu developers propose even GlusterFS improvement ideas here?

### ISSUE THREAD

**Issue by amarts** [kadalu/rfcs#11]
GlusterFS follows [glusterfs-specs](https://github.com/gluster/glusterfs-specs) to get proposal for new ideas (Historically). But in recent times, the ideas were proposed on github and the enhancements were agreed upon, which again is not bad.

But just with 2 RFCs for kadalu projects, and a very successful process, which is followed in other communities as-well, I am inclined for developers to propose a RFC here (even if it would have direct impact on glusterfs), and once discussed, propose it in gluster github, and take it further. This way, at least the ideas proposed from the kadalu community to gluster would be in more convincing manner to gluster community.

As both communities are open-source communities, I thought it would be nicer. 

Feedback welcome!

**Comment by aravindavk** [comment]
I agree with you, looks like a great idea to collaborate. My two concerns are

- How will Gluster developers know the discussion is happening here
- Design docs will diverge in two places.

**Comment by amarts** [comment]
> How will Gluster developers know the discussion is happening here

As usual we will tweet about this in our official handle.

> Design docs will diverge in two places.

Nope. If it is the idea kadalu team is proposing, they get it merged here, and propose it to glusterfs. No further discussion in this place once merged, but will have further discussion in glusterfs.

We can consider a update to this RFC here, with changes done after approval in glusterfs community. But that is optional. The whole idea is, encouraging everyone from kadalu team/community to think of design first, and get consensuses first before coding.

This would help us to propose better plans to community, which anyways has better chance of acceptance. 

Key here is, at any point, we should have both the things open to discussion. It should be completed for review here, and then move to glusterfs repo.

**Comment by aravindavk** [comment]
Makes sense. Thanks for the clarification. I like the final presentation of accepted proposals in Kadalu(https://kadalu.io/rfcs)
